.github/CODEOWNERS

```diff
@@ -493,6 +493,7 @@
 /types/bit-twiddle/                                     @adamzerella
 /types/bitauth/                                         @nitsujlangston
 /types/bitcore-lib/                                     @lautarodragan
+/types/bitcore-lib-cash/                                @jcramer @blockparty-sh @rkalis @bitjson
```

Comment:
this will notify all four accounts, but the bot will only treat @jcramer as an owner because there's only one owner in `Definitions By`. Do you want to add the humans from this list to the `Definitions By` list? You can use multiple lines.

Reply:
Thank you for the review, I have added the others to the `Definitions By` list.